### PR TITLE
Feature/redesign style improvements

### PR DESCRIPTION
**Pull request by nadezhda-serafimova**
Related to: https://github.com/Bratanov/community-driven-radio/pull/37

- Update icon set
- Update logo as use one logo for all themes
- Update main font
- Fix headings, subheadings and songs names sizing
- Fix users count bug overlapping
- Fix player and chat boxes height
- Add fancy hover effects on btns
- Fit view on desktop screen

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -59,9 +60,10 @@
               <!-- t-message || t-system-message -->
             </div>
             <form action="#" class="c-message-form">
-              <input type="textarea" id="message-input" class="c-input-oval" placeholder="Type your message here...">
```
Comment (IvayloAtanasov) [inline public/index.html]: Let's remove the oval class then?

**Comment by IvayloAtanasov** [comment]
Awesome :sunny: 

Congratulations on your first PR :stuck_out_tongue: 

New icons look cool. They are a free set, right? The only problem I see now is now that we set them as background image, we no longer have control over color change on hover. I've saw that you play with opacity, but maybe we can do better? Also we may drop svg injection library if we're switching to setting them as background-image. We should make a discussion about this.

**Comment by nadezhda-serafimova** [comment]
Thank you!

I have explained when we prefer icons to be in the code and when as background images. You're right about the svg color flexibility, but they are not part of the content, so their place is in the style sheet. Let's think about new cool hover effect and drop the svg injection library.

Some other issues we have to fix:
- theme customizing box positioning
- "3 user online" text positioning to be absolute
- smaller font-size in songs lists
- the four sections height and positioning

**Comment by IvayloAtanasov** [comment]
All righty, I'm on it :rocket:

File: public/index.html (hunk 1 above)
Comment (IvayloAtanasov) [reply public/index.html]: Whoops, we still use it in the name modal. :)

Comment (nadezhda-serafimova) [reply public/index.html]: Let's change it here too.

Comment (IvayloAtanasov) [reply public/index.html]: Ummm, change what? You want the input to be styled as oval (as in initial design)?

**Comment by IvayloAtanasov** [comment]
Nade, you can merge at will into the main feature branch, I'm pretty much done here.

Things I've changed:

- I've turned back input hover effect with a little transition. It's nice to have something happening when user selects an input.

- I've managed to add svg icons as mask over background color (essentially changing the color of a given svg). Since masks are not widely supported we have a fallback for older browsers, which will add them with default colors, without any hover effects. Svg injector library is removed.

- Some fixes on footer controls - hover effects, changes in tooltips to prevent vertical scroll.

- Smaller font sizes for song lists. Just FYI, I've also tried to place "..." on longer headings with [text overflow ellipsis](https://www.w3schools.com/cssref/css3_pr_text-overflow.asp), but with no success as it requires fixed width rows.

Sooo,
the only thing left in the list is equal row height for screen and chat. I have an idea how can we achieve that. I've used [this trick](https://stackoverflow.com/questions/1495407/maintain-the-aspect-ratio-of-a-div-with-css) in order to keep the aspect ratio of the imac svg relative to its width. We may try the same thing for the chat, but let's keep that for the other branch. We should also tweak margins to fit everything in one page.

**Comment by IvayloAtanasov** [comment]
I'll play a bit with alignment, but I'm not promising anything pretty :)

**Comment by IvayloAtanasov** [comment]
Hello
...
it's me
...
I was wondering if after all these years you'd like to meet... (or just merge the PR.)


Now straight to the point. I've made some improvements on "full screen alignment with no scroll" issue. I keep thinking that stuffing all things inside a screen will make the content really microscopic. I've reached to a point where it all fits for 1440x850 screens (with no songs in queue) and ... it seems fine for me. If you want you can keep tweaking with it, I'm not going to commit anymore (this time for real :smile:)
Here's a screenshot:
image

As a bonus, we now have change name link in the theme controls popup.

**Review by Bratanov — APPROVED** [review]
Looks a lot cleaner with the new icons and more compact, to a point that my "big middle margin taking screen space" concern is not an issue anymore. Great job @IvayloAtanasov @nadezhda-devlabs 🥇 
If none of you have anything left TODO here we can merge and see about wrapping up #37 🚀🦃

**Pull request merged**